ptp/ptp4u/drain/file_test.go

```diff
@@ -0,0 +1,18 @@
+package drain
+
+import (
+	"io/ioutil"
+	"os"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestCheck(t *testing.T) {
+	file, err := ioutil.TempFile("", "")
+	require.NoError(t, err)
+	defer os.Remove(file.Name())
+
+	check := &FileDrain{FileName: file.Name()}
+	require.True(t, check.Check())
```

Comment:
Could you please also cover “false” situation by specifying a non-existent file name?